### PR TITLE
Add function to distribute token to Audius followers

### PR DESCRIPTION
**Pull request by toshiakitakase**
- Getting Followers to Login to Audius
- Determining the amount of tokens distributed to followers

Requests for contracts are not yet implemented.

### CODE REVIEW

File: frontend/packages/react-app/src/components/organisms/AudiusDistribution.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            >
+              <Text mr={2}>{follower.handle}</Text>
+              <Text>{follower.wallet}</Text>
+            </Flex>
+          )}
+          <Box mt={4}>
+            <Button size="small" variant="danger" onClick={audiusSignOut}>Signout from Audius</Button>
+          </Box>
+        </Box>
+      }
```
Comment (wild-mouse) [inline frontend/packages/react-app/src/components/organisms/AudiusDistribution.jsx:73]: I prefer contained elements as other component file when using ternary operator for readability

**Comment by wild-mouse** [comment]
When I went audius setting page, token amount input form is appeared, but few seconds later form change to login form.
It seems better showing login form first than above behavior.

**Comment by wild-mouse** [comment]
Adding followers success but after closing alert following error occurs:
> Unhandled Rejection (Error): resolver or addr is not configured for ENS name (argument="name", value="", code=INVALID_ARGUMENT, version=contracts/5.0.5)

File: frontend/packages/react-app/src/components/organisms/AudiusDistribution.jsx (hunk 1 above)
Comment (toshiakitakase) [reply frontend/packages/react-app/src/components/organisms/AudiusDistribution.jsx:73]: Fixed!

**Comment by toshiakitakase** [comment]
> When I went audius setting page, token amount input form is appeared, but few seconds later form change to login form.
> It seems better showing login form first than above behavior.

I have changed the order of display.

**Comment by toshiakitakase** [comment]
> Adding followers success but after closing alert following error occurs:

This is an error related to the wallet connection, so I'll consider it in another issue.

**Review by wild-mouse — APPROVED** [review]
LGTM 🏃‍♂️

**Pull request merged**